qa/rpc-tests/schnorrsig.py

```diff
@@ -48,10 +51,28 @@ def setup_network(self, split=False):
         self.is_network_split = False
         self.sync_blocks()
 
-    def run_test(self):
-        self.sync_blocks()
-        self.nodes[0].generate(1)
-
+    def pubkeySearch(self):
+        print("pid: ", os.getpid())
+        privkey = cashlib.randombytes(32)
+        pubkey = schnorr.getpubkey(privkey, compressed=True)
+        lens = array.array("Q",[0 for i in range (0,101)])
+        data = 1
+        while 1:
+            databytes = struct.pack(">Q", data)
```

Comment:
are you actually using this function?..i don't see where it breaks from the loop?

Reply:
I am not calling this as part of the QA test.  This is an infinitely-runing loop that generates lots of signatures.  It can be used to exercise the signature code or to search for different length ECDSA signatures.